packages/frontend/src/utils/getVolunteers.ts

```diff
@@ -0,0 +1,27 @@
+import { FirebaseCollection } from 'uva-shared';
+import { firebaseDb } from 'src/main';
+import { IVolunteer } from 'uva-shared/src/types/volunteer';
+import {
+  collection,
+  getDocs,
+} from 'firebase/firestore';
+
+export const getVolunteers = async (name?: string): Promise<IVolunteer[]> => {
+  try {
+    const volunteersSnapshot = await getDocs(collection(firebaseDb, FirebaseCollection.Volunteers));
```

Comment:
please, use different approach. There with be thousands of entries in volunteers table, so approach "get all and then search in JS" will work poorly. It works for coordinators because there will be less than hundred of them, but not with volunteers.

Please, create index for first name and last name in the table, and then make query request

Reply:
also if name is not specified, there should be pagination